src/ast.rs

```diff
@@ -744,6 +744,12 @@ pub enum TypeQualifier {
     ///
     /// `__volatile`, `__volatile__` (GNU extension)
     Volatile,
+    /// '_Nonnull' (Clang extension)
+    Nonnull,
+    /// '_Null_unspecified' (Clang extension)
+    NullUnspecified,
+    /// '_Nullable' (Clang extension)
+    Nullable,
```

Comment:
Could you please add links to Clang documentation like [here](https://github.com/vickenty/lang-c/blob/master/src/ast.rs#L79)?